docs/setup-sensor/firewalla.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Install Orb on Firewalla
+shortTitle: Firewalla
+metaDescription: Install the Orb sensor on your Firewalla device for network monitoring.
+section: setup-sensor
+layout: guides
+subtitle: 'Difficulty: Easy 🧑‍💻'
+---
+
+# Install Orb on Firewalla
+
+## Introduction
+
+This guide will walk you through the process of setting up the Orb sensor on your Firewalla device. Installing the Orb sensor allows you to monitor the network responsiveness and reliability of your network from anywhere in the world using your mobile device or computer.
+
+With the Orb sensor running on your Firewalla, you can:
+
+- Monitor your network connectivity and performance
+- Track network reliability and responsiveness over time
+- Receive push notifications on your Android or iOS device when your Firewalla experiences connectivity issues
+
+## Compatibility
+
+The following Firewalla devices noted as "Supported" support Docker and should be able to run the Orb sensor:
+
+| Device | Supported | Validated | Notes |
+|--------|----------------|-----------|--|
+| Gold | ✓ | | |
+| Gold SE | ✓ | ✓ | |
+| Gold Plus | ✓ | ✓ | |
+| Gold Pro | ✓ | | |
+| Purple | ✓ | | |
+| Purple SE | ✓ | | |
+| Blue Plus | ✓ | | |
+| Blue | ✗ | | |
+| Red | ✗ | | |
+
+The "Validated" column denotes device models that have been verified to run Orb by the Orb Forge team or members of the Orb community.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- A Firewalla device (Gold, Purple, or Blue Plus)
```

Comment:
Perhaps mention the Firewalla app here? It's not necessarily required if you know the SSH password some other way, but AFAIK it's the only way to retrieve the SSH password you will need to complete this.